app/controllers/admin_referrals_controller.rb

```diff
@@ -14,19 +14,20 @@ def create
     @referral = AdminReferral.new(admin_referral_params)
 
 
-    if verify_recaptcha(model: @referral) && @referral.save
+    if @referral.save
+    # if verify_recaptcha(model: @referral) && @referral.save
```

Comment:
Hey Peter is this code obsolete now? Are we validating recaptcha somewhere else?

Reply:
Nope, we have to add these back, they were commented out just for testing purposes,